app/views/hyrax/dashboard/_repository_growth.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="panel-heading">
+  <h3 class="panel-title"><%= t('.title') %></h3>
+  <div><%= t('.subtitle') %></div>
+</div>
+<div class="panel-body text-center">
+  <% repository_growth = @presenter.repository_growth.to_json %>
+  <div id="dashboard-growth" aria-hidden="true" style="height: 200px"><script>
+//<![CDATA[
+Hyrax.statistics.repositoryGrowth = <%== repository_growth %>
+//]]>
+</script></div>
+  <table class="table table-striped sr-only text-left">
```

Comment:
This looks really good.  The only suggestion I'd have to finish it off is aria labeling each table.  For example:

`<table aria-label="<%= t('.repository_growth.subtitle') %>" class="table table-striped sr-only text-left">`

However that translation title sneaks in there...

Reply:
https://www.w3.org/TR/wai-aria-practices/examples/table/table.html